api/src/pcapi/core/finance/backend/__init__.py

```diff
@@ -36,6 +36,16 @@ def get_invoice(reference: str) -> dict | None:
     return backend.get_invoice(reference)
 
 
+def get_time_to_sleep_between_2_sync_requests() -> int:
```

Comment:
```suggestion
def get_time_to_sleep_between_two_sync_requests() -> int:
```
Pfff, cette génération texto, qui veut pas écrire correctement :p